File: blog/snippets/javascript/fizz_buzz.js
// FizzBuzz implementation with reversal of string

function reverseString(str) {
  return str.split('').reverse().join('');
}

for (let num = 1; num < 101; num++) {
  if (num % 3 == 0 && num % 5 == 0) {
    console.log(reverseString('zzubzziF'));
  }
  if (num % 3 == 0) {
    console.log(reverseString('zziF'));
  }
  if (num % 5 == 0) {
    console.log(reverseString('zzuB'));
  }
  else {
    console.log(num);
  }
}

File: blog/snippets/python/fizz_buzz.py
# FizzBuzz implementation with reversal of string

for num in range(1, 101):
    if num % 3 == 0 and num % 5 == 0:
        print('zzuBzziF'[::-1])
    elif num % 3 == 0:
        print('zziF'[::-1])
    elif num % 5 == 0:
        print('zzuB'[::-1])
    else:
        print(num)
